File: js/findTheBomb.js
// Create a function that finds the word "bomb" in the given string. If found, return "Duck!!!",
// otherwise return "There is no bomb, relax.".
// https://edabit.com/challenge/r4hycBjiBNMs6AAjq

const bomb = (str) =>
  str.toLowerCase().includes("bomb") ? "Duck!!!" : "There is no bomb, relax.";

module.exports = bomb;

File: CSharp/FindTheBomb.cs
namespace CSharp
{
    // Create a function that finds the word "bomb" in the given string. If found, return "Duck!!!", 
    // otherwise, return "There is no bomb, relax.".
    // https://edabit.com/challenge/JYEufqRvkusjr5R58
    public static class FindTheBomb
    {
        public static string Bomb(string txt) => txt.ToLower().Contains("bomb") ? "Duck!!!" : "There is no bomb, relax.";
    }
}